db/import_listener.go

```diff
@@ -152,7 +152,10 @@ func (il *importListener) Stop() {
 			// Close open PIndexes before stopping the manager.
 			_, pindexes := il.cbgtContext.Manager.CurrentMaps()
 			for _, pIndex := range pindexes {
-				il.cbgtContext.Manager.ClosePIndex(pIndex)
+				err := il.cbgtContext.Manager.ClosePIndex(pIndex)
+				if err != nil {
+					base.Debugf(base.KeyImport, "Error closing primary index: %v", err)
```

Comment:
This is a partitioned index, but might as well keep as pindex as that's what it is referred to throughout.

```suggestion
					base.Debugf(base.KeyImport, "Error closing pindex: %v", err)
```